docker/nginx/nginx.conf

```diff
@@ -3,80 +3,73 @@ worker_processes 4;
 events { worker_connections 1024; }
 
 http {
-	upstream app_server {
-        	server app-1:3000 fail_timeout=5s max_fails=5;
-	}
-
-	server {
-	    listen 80 default_server;
-	    listen [::]:80 default_server;
-	
-	    charset utf-8;
-	
-	    access_log /var/log/nginx/openhab.org-access.log;
-	    error_log /var/log/nginx/openhab.org-error.log;
-	    client_max_body_size 300m;
 	
-	    root /opt/openhabcloud;
+	include /etc/nginx/mime.types;
```

Comment:
for the record: That was exactly what I meant :P Probably I haven't said it as clear as possible :)